docs: tweak quick start (#1746)
GreptimeTeam/docs: docs/getting-started/quick-start.md

psql -h 127.0.0.1 -p 4003 -d public
```

> You can also use your browser to access the built-in DB dashboard at `http://{db_host}:4000/dashboard` to run the SQL queries in this document.

Reviewer: ```suggestion
You can also use your browser to access the built-in DB dashboard at `http://127.0.0.1:4000/dashboard` to run the SQL queries in this document.
```

## Create tables

> Suppose you have a wide event table named `grpc_latencies` that stores the gRPC services and processing latencies of your application.

Reviewer: the table is not wide at all..

Author: Ok. Let's say it's an event table.

## Create tables

Suppose you have a wide event table named `grpc_latencies` that stores the gRPC latencies of your application.
The table schema is as follows:

```sql
CREATE TABLE grpc_latencies (
  ts TIMESTAMP TIME INDEX,
  host STRING INVERTED INDEX,
  method_name STRING,
  latency DOUBLE,
  PRIMARY KEY (host, method_name)
) with('append_mode'='true');
```

- `ts`: The timestamp when the metric was collected. It is the time index column.

> - `host`: The hostname of the application server, enabling [inverted index](/user-guide/manage-data/data-index#inverted-index).

Reviewer 2: ```suggestion
- `host`: The hostname of the application server, enabling [inverted index](/user-guide/manage-data/data-index.md#inverted-index).
```

- `method_name`: The name of the RPC request method.
- `latency`: The latency of the RPC request.

> 
> And it's append only.

Reviewer 2: ```suggestion
- Set `append_mode` to true to preserve duplicate rows.
```

Additionally, there is a table `app_logs` for storing application logs:

```sql
CREATE TABLE app_logs (
  ts TIMESTAMP TIME INDEX,
  host STRING INVERTED INDEX,
  api_path STRING,
  log_level STRING,
  log_msg STRING FULLTEXT INDEX WITH('case_sensitive' = 'false'),
  PRIMARY KEY (host, log_level)
) with('append_mode'='true');
```

- `ts`: The timestamp of the log entry. It is the time index column.
- `host`: The hostname of the application server, enabling inverted index.
- `api_path`: The API path.
- `log_level`: The log level of the log entry.

> - `log_msg`: The log message, enabling [fulltext index](/user-guide/manage-data/data-index#fulltext-index).

Reviewer 2: ```suggestion
- `log_msg`: The log message, enabling [fulltext index](/user-guide/manage-data/data-index.md#fulltext-index).
```

>  It's append only, too.

Reviewer 2: ```suggestion
- Set `append_mode` to true to preserve duplicate rows.
```

Author: To preserve duplicate rows is not the purpose but the side effect. Append only is for performance.

 ::::tip

> We use SQL to ingest the data below, so we need to create the tables manually. However, GreptimeDB is [schemaless](/user-guide/ingest-data/overview/#automatic-schema-generation) and can automatically generate schemas when using other ingestion methods.

Reviewer 2: ```suggestion
We use SQL to ingest the data below, so we need to create the tables manually. However, GreptimeDB is [schemaless](/user-guide/ingest-data/overview.md#automatic-schema-generation) and can automatically generate schemas when using other ingestion methods.
```

2 rows in set (0.11 sec)
```

### Search logs by keyword

Filter the log messages by keyword `timeout`:
```sql
SELECT * FROM app_logs WHERE lower(log_msg) @@ 'timeout' AND ts > '2024-07-11 20:00:00';
```

```sql
+---------------------+-------+------------------+-----------+--------------------+
| ts                  | host  | api_path         | log_level | log_msg            |
+---------------------+-------+------------------+-----------+--------------------+
| 2024-07-11 20:00:10 | host1 | /api/v1/billings | ERROR     | Connection timeout |
| 2024-07-11 20:00:10 | host1 | /api/v1/resource | ERROR     | Connection timeout |
| 2024-07-11 20:00:14 | host1 | /api/v1/billings | ERROR     | Timeout            |
| 2024-07-11 20:00:14 | host1 | /api/v1/resource | ERROR     | Timeout            |
+---------------------+-------+------------------+-----------+--------------------+
```

> 

Reviewer 2: It is recommended to add a link to `/user-guide/logs/query-logs.md#simple-term-matching`
